src/Runtime/Monad.hs

```diff
@@ -81,7 +81,7 @@ extScope env = local (modifyEval (env++))
 -- | Lookup a name in the environment FIXME
 lookupName :: Name -> Eval (Maybe Val)
 lookupName n = do
-  env <- (evalEnv <$> ask)
+  env <- getEnv
```

Comment:
I'm in favor of cleaning up these tiny issues as we come across them. Good catch!